### ISSUE TITLE
Better accessibility on Input component

### ISSUE THREAD

**Issue by agneym** [bigbinary/neeto-ui#239]
* Input should contain `required=''` if field is required
* Label should be associated to input by `id`
* Error text and Help text should be associated to input by `aria-describedby`
* `aria-invalid` should be set to `true` on input if invalid 
* decorations like error icon and * should be set to `aria-hidden` so they aren't read out
